docs/competition/volunteers/refreshments.md

```diff
@@ -0,0 +1,28 @@
+# Refreshments
+
+It's important that volunteers stay hydrated, which can be difficult for those in roles which don't allow them to move. (Don't give refreshments to competitors).
+
+**In SR2019, the refreshments are in the Volunteer Rest room, which is past the toilets and to the left. Also Water points are on both level 2 and 3 in the corner furthest from the toilets.**
+
+It is the responsibility of the Refreshments role to offer and provide refreshments to all volunteers, especially those who can't leave their roles.
+
+To do this, you should grab a tray, fill it with a mix of drinks and food, and go to all volunteer positions, offering refreshments.
+Please also fill some cups with water from a nearby water fountain, and brew coffee/tea if available.
+
+Please don't forget about those at tech decks, those at reception/helpdesk, and those who are commentating/marshalling.
```

Comment:
Could we be explicit about why this is the case?
```suggestion
People at the following locations will be unable to leave their posts to fetch refreshments for themselves, so it is important to take refreshments to them regularly:
- tech desks
- reception
- helpdesk
- arena: commentator, marshals, head shepherd, score entry

The tech desks roles and commentator are particularly important as they do not rotate between people at all.
```

Reply:
Thanks for the fix!